src/test/java/core/basesyntax/cvswork/FileReaderImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.cvswork;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitTransactionService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private List<Fruit> fruits = new ArrayList();
+    private FruitTransactionService manipulation;
+    private List<String> fruitTransactions = new ArrayList();
+    private List<String> emptyFile = new ArrayList();
+    private List<String> transactions = new ArrayList();
+
+    @Before
+    public void setUp() {
+        FileReader read = new FileReaderImpl();
+        transactions = read.read("src/test/java/core/basesyntax/resourse/withoutName.cvs");
+        emptyFile = read.read("src/test/java/core/basesyntax/resourse/emptyLine.cvs");
+        fruitTransactions = read.read("src/test/java/core/basesyntax/resourse/normalFile.cvs");
+
+    }
+
+    @Test
+    public void is_valid_null_List() {
```

Comment:
rename your test methods, pattern is `(name of the method that is testes)_(special event that happens in this method)_(ok / notOk behavior)`, for example, read_emptyFIle_ok